spec/index.bs

```diff
@@ -3312,6 +3330,7 @@ Given a {{GPUCommandEncoder}} |encoder| and the arguments {{GPUBufferCopyView}}
   For |source|:
   - |source| must be [=valid=].
   - |source|.{{GPUBufferCopyView/buffer}}.{{GPUBuffer/[[usage]]}} must contain {{GPUBufferUsage/COPY_SRC}}.
+  - |source|.{{GPUTextureDataLayout/bytesPerRow}} must be a multiple of 256.
```

Comment:
Shouldn't this point to `GPUBufferCopyView Valid Usage`?

Reply:
I second this, but also maybe this rule should be a part of a new "GPUTextureDataLayout Valid Usage" and then "GPUBufferCopyView Valid Usage" just adds the valid buffer check?

Reply:
`GPUTextureDataLayout` is also used in places where there is no such requirement, hence I specifically left the requirement associated with the `GPUBufferCopyView` and the copy operations.
I refactored this to refer to the valid usage rules defined, I think it's better now :)